scripts/release_ppa.sh

```diff
@@ -76,7 +76,7 @@ mv solidity solc
 
 # Fetch jsoncpp dependency
 mkdir -p ./solc/deps/downloads/ 2>/dev/null || true
-wget -O ./solc/deps/downloads/jsoncpp-1.7.7.tar.gz https://github.com/open-source-parsers/jsoncpp/archive/1.7.7.tar.gz
+wget -O ./solc/deps/downloads/jsoncpp-1.8.4.tar.gz https://github.com/open-source-parsers/jsoncpp/archive/1.8.4.tar.gz
```

Comment:
@chriseth why does the ppa script needs to download this separately? I thought cmake will do that on its own?

Reply:
Because you cannot download anything in PPA builds. So you have to prepare the sources yourself. If you put the archive in the place where CMake would download it, it will only check the hash and unzip it from there.

Reply:
Not even with `cmake`? :)

Reply:
What I heard is the internet access is restricted only to official packages repo.

Reply:
You have no internet access at all. You can declare dependencies on other packages, that's it, and I think this is a good thing: Building a package should be a deterministic, offline process.